lambda/index.py

```diff
@@ -337,6 +345,9 @@ def main(event, context):
     delivery_role = os.environ['DELIVERY_STREAM_ROLE_ARN']
     timeout = os.environ['TIMEOUT']
 
+    global ignore_delete_errors
+    ignore_delete_errors = os.environ['IGNORE_DELETE_ERRORS']
```

Comment:
Set the environment variable directly when declaring `ignore_delete_errors` (line 32).